src/main/twirl/views/manage/whatWeWillAskPreview.scala.html

```diff
@@ -0,0 +1,57 @@
+@(backUrl: String, opp: Opportunity, sectionNumber: Int, appForm: ApplicationForm)
+
+@currentSection = @{
+    opp.description.find(_.sectionNumber == sectionNumber)
+}
+
+@main(s"Opportunity: ${opp.title} - RIFS", backLink = Some(BackLink("Opportunity template",controllers.manage.routes.OpportunityController.showOverviewPage(opp.id).url)), displayUserName = Some("Portfolio Peter")) {
+
+    <div class="rifs-heading-panel">
+        <p>This section is automatically generated from the questions on the application form</p>
+    </div>
+
+    <div class="grid-row">
+        <div class="column-two-thirds">
+
+
+            <header class="page-header">
+                <h1 class="heading-xlarge">@opp.title</h1>
+            </header>
+
+            @currentSection.map { s =>
+                <article>
+                    <h2 class="heading-large">@s.sectionNumber. @s.title</h2>
+                    <div class="text">
+                    @if(s.sectionType == OppSectionType.Questions) {
+                        @helpers.textForQuestions(appForm)
+                    } else {
+                        @s.text.map(_.split("\n\n")).map { ps =>
+                            @ps.map { p =>
+                                <p class="text">@p</p>
```

Comment:
I'm not sure where the "wordcount: " element is coming from, but it would be nice if the wordcount could be budged up to be on a newline just beneath the section.  Otherwise fine :)